tmt/plugins/__init__.py

```diff
@@ -32,16 +43,24 @@ def explore() -> None:
     try:
         paths = [
             os.path.realpath(os.path.expandvars(os.path.expanduser(path)))
-            for path in os.environ["TMT_PLUGINS"].split(os.pathsep)]
+            for path in os.environ[ENVIRONMENT_NAME].split(os.pathsep)]
     except KeyError:
-        log.debug('No custom plugin locations detected in TMT_PLUGINS.')
+        log.debug(f'No custom plugin locations detected in {ENVIRONMENT_NAME}.')
         paths = []
     for path in paths:
         for module in discover(path):
             if path not in sys.path:
                 sys.path.insert(0, path)
             import_(module, path)
 
+    # Import by entry_points
+    try:
+        for found in entry_points()[ENTRY_POINT_NAME]:
+            log.debug(f'Loading plugin from: {found}')
+            found.load()
+    except KeyError:
+        pass
```

Comment:
I'd probably add a message similar to `log.debug(f'No custom plugin locations detected in {ENVIRONMENT_NAME}.')`

Reply:
The only problem is that such debug message will be printed each time while ENVIRONMENT_NAME is printed only if you exported such variable but there is nothing in its directories.

Reply:
Isn't it logged every time, especially when the envvar is left unset? `os.environ[ENVIRONMENT_NAME]` should raise a `KeyError` in such a case, leading to the debug message.

Anyway, I hate silence, therefore I for one would slap a debug message at that `except` branch even if it'd be printed every time `tmt` runs. It's a debug-level after all.

Reply:
Ah, right. Not enough coffee yet.
 Does it print at all though? I couldn't come up with `tmt -ddd` which would do the print with debug level already set.

Either way I've changing it to:
If nothing is in entry_point: `No custom plugins detected for 'tmt.plugin'.`
If founds something  `Loading plugin from: EntryPoint(name='custom_name', value='module:class', group='tmt.plugin')`

Should it be e.g `Loading plugin "module:class"` instead? ("custom_name" is ignored)

Reply:
I wouldn't mind including `custom_name`, it might be helpful when debugging issues.